AvoidSinglePipeOperator: allow single pipe operator with multiline argument

webwarrior-ws opened the pull request:
Do not apply this rule when an argument passed to pipe operator spans multiple lines.

knocte commented on src/FSharpLint.Core/Rules/Conventions/AvoidSinglePipeOperator.fs:
```diff
@@ -8,6 +8,9 @@ open FSharp.Compiler.Syntax
 open FSharpLint.Framework.Rules
 
+let isSingleLine (range: FSharp.Compiler.Text.range) =
+    range.EndLine  = range.StartLine
```
@webwarrior-ws please squash 2 spaces into 1

webwarrior-ws commented:
> * Please squash 2nd with 3rd commit, no reason to separate them.
> * Please avoid ending dots in commit msg titles.

Squashed last 2 commits and fixed commit message title.
